modules/rostests/apitests/shell32/ExtractIconEx.cpp

```diff
@@ -22,16 +22,33 @@ EXTRACTICONTESTS IconTests[] =
     {L"%SystemRoot%\\System32\\autochk.exe", 0},
 
     /* Non-existing files */
-    {L"%SystemRoot%\\non-existent-file.sdf", 0}
+    {L"%SystemRoot%\\non-existent-file.sdf", 0},
+
+    /* Multiple icons in the same ICO file (6 icons) */
+    {L"%SystemRoot%\\bin\\sysicon.ico", 1},
+
+    /* Multiple icons in the same EXE file (18 icons) */
+    {L"%SystemRoot%\\explorer.exe", 18},
```

Comment:
Are the explorer.exe tests compatible to WinXP/Win2k3?

Reply:
@katahiromz, Yes, they are compatible with WinXP/Win2k3. I have attached screen shots in the Description above to demonstrate. I did have to add the two ROS.ico and sysicon.ico to the C:\Windows\bin directory first though.

Reply:
Note: Explorer.exe in WinXP/Win2k3 has 18 embedded icons just like the one in ReactOS.